src/main.js

```diff
@@ -0,0 +1,315 @@
+import './style.css';
+import Form from './form';
+import ToDo from './todo';
+import FormForProject from './formForProject';
+import Project from './project';
+
+const Main = (() => {
+  // load localStorage
+  let projects = JSON.parse(localStorage.getItem('projects'));
+  let lastProject = JSON.parse(localStorage.getItem('lastProject'));
+
+  if (projects == null || projects.length === 0) {
+    projects = [new Project('Your first project', [])];
+  } else {
+    const dummy = [...projects];
+    projects = [];
+    for (let i = 0; i < dummy.length; i += 1) {
+      const timmy = [];
+      dummy[i].toDo.forEach((todo) => {
+        timmy.push(
+          new ToDo(
+            todo.title,
+            todo.description,
+            todo.dueDate,
+            todo.priority,
+            todo.check,
+          ),
+        );
+      });
+      projects.push(new Project(dummy[i].title, timmy));
+    }
+  }
+
+  if (
+    projects.filter((project) => project.title === lastProject).length === 0
+  ) {
+    lastProject = projects[0].title;
+  }
+
+  // save last project
+
+  const saveLastProject = (project) => {
+    localStorage.setItem('lastProject', JSON.stringify(project.title));
+  };
+
+  // validates project is filled out
+
+  const validateProject = () => {
+    const form = document.getElementById('project-form');
+    if (form[0].value === '') {
+      return alert('Title must be filled out'); // eslint-disable-line no-alert
+    }
+    const project = new Project(form[0].value);
+
+    projects.push(project);
+    return project.save(projects);
+  };
+
+  // validates ToDo and creates a new one
+
+  const validateToDo = (project = null, toDo = null) => {
+    const form = document.getElementById('todo-form');
+    if (
+      form[0].value === ''
+      || form[1].value === ''
+      || form[2].value === ''
+      || form[3].value === ''
+    ) {
+      return alert('Form should be completely filled out.'); // eslint-disable-line no-alert
```

Comment:
This should be production code, we shouldn't have `alert`, please remove the `alert`.

Reply:
Done! Thanks for pointing it out, we changed the alert for a "required" attribute for each form field